### ISSUE TITLE
Detailed Hyperparameter Settings for CIFAR-10 10IPC/50IPC Experiments

### ISSUE THREAD

**Issue by DingFong** [DataDistillation/DataDAM#6]
Hello, I am trying to reproduce the CIFAR-10 10IPC/50IPC experiments based on the paper. Could you please provide detailed hyperparameter settings as I'm currently facing challenges in achieving the expected test performance? Thank you!

My setting of CIFAR-10 (10IPC):
batch_real/ batch_train: 256
model: ConvNetD3
epoch_eval_train: 1800
Iteration: 8000
lr_img: 1
lr_net: 0.01
init: real
dsa_strategy: color_crop_cutout_flip_scale_rotate
zca: True
task_balance: 0.01
power in equation 1.: 4

My setting of CIFAR-10 (50IPC):
batch_real/ batch_train: 256
model: ConvNetD3
epoch_eval_train: 1800
Iteration: 8000
lr_img: 1
lr_net: 0.01
init: real
dsa_strategy: color_crop_cutout_flip_scale_rotate
zca: True
task_balance: 0.01
power in equation 1.: 4

**Comment by DataDistillation** [comment]
Thank you for your interest in our work. We recommend that you initialize your synthetic data using the associated images and labels present in the Initial_Synthetic_Dataset Folder.

**Comment by DingFong** [comment]
Thank you for your prompt response. I appreciate your clarification. However, to ensure I'm following the experiment details accurately, could you confirm whether the model's normalization is implemented as instance normalization or batch normalization? According to the paper, it seems to be instance normalization, but I've encountered problem in achieving the expected results in this setting. Any insights or suggestions would be valuable. Thank you for your assistance!

**Comment by DataDistillation** [comment]
Thanks for asking, we are using Batch Normalization -- we have set our default configuration in the parser arguments of the main file. Our default networks are described in utils.py and networks.py

**Comment by Byunggwan-Son** [comment]
@DingFong Have you reached the expected results? I am currently having a problem to reproduce the result for the CIFAR-10 1 IPC setting.
